.bluemix/pipeline.yml

```diff
@@ -74,6 +80,31 @@ stages:
         echo "https://www.ibmwatsonconversation.com"
         exit 1
       fi
+      
+      # Tracking Metrics
+      if [ "yes" != "$(echo "$DISABLE_METRIC" | awk '{print tolower($0)}')" ]; then
```

Comment:
so as it stands now it's enabled by default

Reply:
we should echo out some stuff here...

         echo "============================================================="
         echo "Adding deployment data to metrics tracker... "
         echo "============================================================="

and then again at the end if it was successful uploaded. I say this because I'm not seeing anything in the logs past the following:

```
==================================================
Created new workspace: WORKSPACE_ID: 645bdf74-c71a-4693-86ab-a860bb3eed1e
==================================================
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   310    0   310    0     0   2832      0 --:--:-- --:--:-- --:--:--  2844
{
  "name": "watson-banking-chatbot",
  "created": "2017-11-07T05:19:42.239Z",
  "updated": "2017-11-07T05:19:42.239Z",
  "language": "en",
  "metadata": {
    "runtime_version": "2016-09-20"
  },
  "description": "A cognitive banking chatbot",
  "workspace_id": "645bdf74-c71a-4693-86ab-a860bb3eed1e",
  "learning_opt_out": false,
  "status": "Training"
}

Finished: SUCCESS
```